# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/webidl/moz.build
# Commit: 9c8b6e35bf3f
# Full Hash: 9c8b6e35bf3f9916e114ced0473be4b778c7bab1
# Author: Andrea Marchesini <amarchesini@mozilla.com>
# Date: 2024-09-11 09:21:34
# Regressor Bug: 1475599
# File Overlap Count: 1
# Description:
#   Bug 1475599 - part 1 - CookieStore API - WebIDL, r=smaug,edgul
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D215144
# ==============================================================================

diff -r a9b3b52f33a5 -r 9c8b6e35bf3f dom/webidl/moz.build
--- a/dom/webidl/moz.build	Tue Sep 10 16:21:33 2024 +0000
+++ b/dom/webidl/moz.build	Tue Sep 10 16:31:38 2024 +0000
@@ -480,6 +480,8 @@
     "Console.webidl",
     "ConstantSourceNode.webidl",
     "ConvolverNode.webidl",
+    "CookieChangeEvent.webidl",
+    "CookieStore.webidl",
     "CreateOfferRequest.webidl",
     "CredentialManagement.webidl",
     "Crypto.webidl",